### ISSUE TITLE
Option to serve compressed responses

### ISSUE THREAD

**Issue by Lusitaniae** [fluent/fluent-plugin-prometheus#219]
I'm currently using fluentd to generate a giant amount of metrics which have already passed 50Mb response sizes in some of our nodes

I can bump limits and timeouts on the scraping side of things, but compressing the response would probably be very useful here! 

```
$ curl -s http://node1:9106/metrics | wc -c
37420840
$ curl -s http://node1:9106/metrics | gzip -c | wc -c
1877477
```

**Comment by Athishpranav2003** [comment]
@Lusitaniae can you close this issue?

**Comment by ashie** [comment]
Fixed via #223

**Comment by ashie** [comment]
I've released v2.2.0 that supports gzip.
